src/pyhf/tensor/jax_backend.py

```diff
@@ -42,8 +43,19 @@ class jax_backend(object):
 
     def __init__(self, **kwargs):
         self.name = 'jax'
+        self._mode = 'GPU'
+        os.environ["JAX_PLATFORM_NAME"] = self._mode.lower()
         config.update('jax_enable_x64', True)
 
+    @property
+    def mode(self):
+        return self._mode
+
+    @mode.setter
```

Comment:
To make this easier to deal with -- we shouldn't allow the mode to be changed after initialization. If the user wants to change the mode, they should supply a new backend...